orga/app/components/routes/authenticated/campaign/assessment/results.hbs

```diff
@@ -21,7 +21,7 @@
           </td>
           <td>
             <ProgressionGauge @total={{competenceResult.totalSkillsCountPercentage}} @value={{competenceResult.validatedSkillsCountPercentage}}>
-              <p class="sr-only">Ce participant a validé </p>{{competenceResult.validatedSkillsCountPercentage}}%<p class="sr-only"> de la compétence {{competenceResult.name}}.</p>
+              <p class="sr-only">{{t 'pages.assessment-individual-results.table.column.results.explanation-begin'}}</p>{{competenceResult.validatedSkillsCountPercentage}}%<p class="sr-only">{{t 'pages.assessment-individual-results.table.column.results.explanation-end' competence=competenceResult.name}}</p>
             </ProgressionGauge>
```

Comment:
**suggestion (non-blocking) :** On pourrait faire  un span sr-only avec toute la phrase `This participant has validated  X% of the skill Resolving technical problems.`  et afficher les % dans la page avec aria-hidden=true

Reply:
oui aussi, j'y ai pensé, mais aucune des solutions n'est satisfaisante je trouve :/

Reply:
Remplacer les `p` par des `span` suffit pour le screen reader